keel-titus-plugin/src/main/kotlin/com/netflix/spinnaker/keel/titus/batch/ContainerJobConfig.kt

```diff
@@ -1,123 +1,112 @@
 package com.netflix.spinnaker.keel.titus.batch
 
-import com.netflix.spinnaker.keel.titus.batch.ImageVersionReference.*
+import com.fasterxml.jackson.databind.ObjectMapper
+import com.netflix.spinnaker.keel.api.ec2.Capacity
+import com.netflix.spinnaker.keel.api.titus.TitusServerGroup
+import java.time.Duration
 
-const val RUN_JOB_TYPE : String = "runJob"
-
-/**
- * Containers can be identified by either:
- *  tag (e.g., "latest")
- *  digest (e.g., "sha256:780f11bfc03495da29f9e2d25bf55123330715fb494ac27f45c96f808fd2d4c5")
- */
-enum class ImageVersionReference {
-  TAG,
-  DIGEST
-}
+const val RUN_JOB_TYPE: String = "runJob"
 
 /**
  * Holds the config info needed to execute a batch job on Titus via the Run Job stage
  */
 data class ContainerJobConfig(
-  val name : String,
```

Comment:
I removed this because it's just the name of the stage and is defaulted to the stage type